docs/sql-ref-syntax-aux-show-tables.md

```diff
@@ -18,5 +18,86 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
 
-**This page is under construction**
+The `SHOW TABLES` statement returns all the tables for an optionally specified database.
+Additionally, the output of this statement may be filtered by an optional matching
+pattern. If no database is specified then the tables are returned from the 
+current database.
+
+### Syntax
+{% highlight sql %}
+SHOW TABLES [{FROM|IN} database_name] [LIKE 'regex_pattern']
+{% endhighlight %}
+
+### Parameters
+<dl>
+  <dt><code><em>{FROM|IN} database_name</em></code></dt>
+  <dd>
+     Specifies the `database` name from which tables are listed.
+  </dd>
+  <dt><code><em>LIKE 'regex_pattern'</em></code></dt>
+  <dd>
+     Specifies the regex pattern that is used to filter out unwanted tables.
+       <br> - The pattern is a regex except `*` and `|`characters
```

Comment:
Nit: can you use an HTML unordered list? `<ul><li>...` etc? Or just make this a paragraph, not bullet points.

I still think this is kind of unclear, per the last comment. "except `*` ..." should be clarified. "except that `*` and `|` have a different meaning."

Reply:
done.
used the html ul and li  tags  
and added the description like below
 `Except `*` and `|` characters remaining characters will follow the regular expression convention.`